lib/rubocop/cop/style/mutable_constant.rb

```diff
@@ -132,6 +134,13 @@ def frozen_string_literal?(node)
           FROZEN_STRING_LITERAL_TYPES.include?(node.type) && frozen_string_literals_enabled?
         end
 
+        def shareable_constant_value?(node)
+          return false if target_ruby_version < 3.0
+          return false if node.nil?
```

Comment:
Is there a valid reason for this method to ever be called with `node == nil`?

Reply:
All the scenarios like the following https://github.com/rubocop/rubocop/blob/master/spec/rubocop/cop/style/mutable_constant_spec.rb#L28 passes a nil node to this

Reply:
Oh, I see. That's a mistake, fixed in #9931